feed/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Tells docker to use the latest Rust official image
+FROM rust:latest as builder
+WORKDIR /feed
+# Copy the project files from your machine to the container
+COPY ./ ./
+# Build your application for release, inside the container
+RUN cargo build --release
+
+FROM debian:buster-slim
```

Comment:
Why not alpine? Slimmer

Reply:
I'll try it.

Reply:
I tried alpine, doesn't work, the issue is probably with the absence of the necessary libraries ([see this](https://stackoverflow.com/questions/49098753/unable-to-run-a-docker-image-with-a-rust-executable)). `debian:buster-slim` is already only 70mb, don't think it's worth trying to make it slimmer.

Reply:
Although I used slimer `rust:slim-buster` for the builder image instead of `rust:latest`.

Reply:
Okay :-(